pkg/apis/build/v1alpha2/build_pod.go

```diff
@@ -1211,9 +1219,12 @@ func cosignSecretArgs(secret corev1.Secret) []string {
 	return cosignArgs
 }
 
-func envs(envs []corev1.EnvVar, envVar corev1.EnvVar) []corev1.EnvVar {
-	if envVar.Name != "" && envVar.Value != "" {
-		envs = append(envs, envVar)
+func envs(envs []corev1.EnvVar, envVar1 corev1.EnvVar, envVar2 corev1.EnvVar) []corev1.EnvVar {
```

Comment:
Would it be useful to have `envs` accept a variable number of arguments? Something like:
```golang
func envs(envs []corev1.EnvVar, envVars ...corev1.EnvVar) []corev1.EnvVar {
	for _, envVar := range envVars {
		if envVar.Name != "" && envVar.Value != "" {
			envs = append(envs, envVar)
		}
	}
}
```